README.md

```diff
@@ -111,37 +111,37 @@ fun buildCameraEventListener(): CameraEventListener = object : CameraEventListen
+| destroy                      | -                                               | -                                                                               | void        | Remove camera preview.
+| toggleCameraLens             | -                                               | -                                                                               | void        | Set camera lens facing front or back.
+| setCameraLens                | `cameraLens: String`                            | <ul><li>`"front"`</li><li>`"back"`</li></ul>                                    | void        | Set camera to use "front" or "back" lens. Default value is "front".
+| getCameraLens                | -                                               | -                                                                               | Int         | Return `Int` that represents lens face state: 0 for front 1 for back camera.
+| setNumberOfImages            | `numberOfImages: Int`                           | Any positive `Int` value                                                        | void        | Default value is 0. For value 0 is saved infinity images. When saved images reached the "number os images", the `onEndCapture` is triggered.
+| setTimeBetweenImages         | `timeBetweenImages: Long`                       | Any positive number that represent time in milli seconds                        | void        | Set saving face/frame images time interval in milli seconds.
+| setOutputImageWidth          | `width: Int`                                    | Any positive `number` value that represents in pixels                           | void        | Set face image width to be created in pixels.
+| setOutputImageHeight         | `height: Int`                                   | Any positive `number` value that represents in pixels                           | void        | Set face image height to be created in pixels.
+| setSaveImageCaptured         | `enable: Bool`                                  | `true` or `false`                                                               | void        | Set to enable/disable save image when capturing face and frame.
+| setFaceDetectionBox          | `enable: Bool`                                  | `true` or `false`                                                               | void        | Set to show a detection box when face detected.
+| setFaceContoursColor         | `alpha: Int, red: Int, green: Int, blue: Int`   | Positive value between 0 and 255                                                | void        | Set to show/hide face contours when face detected.
+| setFaceContours              | `enable: Bool`                                  | `true` or `false`                                                               | void        | Set face contours ARGB color.
```

Comment:
Move this definition line to 1 above.
This is `setFaceContoursColor` description.